marketplace/commercelayer-products/extension.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "commercelayer-products",
+  "name": "Commerce Layer",
+  "fieldTypes": ["Symbol"],
```

Comment:
The value is not really used if we use app endpoints but I'd still do `["Symbol", "Symbols"]` if we ever decide to handle `AppDefinition` in the tooling.

Reply:
done.